src/Compilers/Core/Portable/SymbolDisplay/SymbolDisplayMiscellaneousOptions.cs

```diff
@@ -72,5 +72,11 @@ public enum SymbolDisplayMiscellaneousOptions
         /// Append '!' to non-nullable reference types.
         /// </summary>
         IncludeNotNullableReferenceTypeModifier = 1 << 8,
+
+        /// <summary>
+        /// Insert a tuple into the display parts as a single part instead of multiple parts (similar
+        /// to how anonymous types are inserted0.
+        /// </summary>
+        DoNotExpandTupleType = 1 << 9,
```

Comment:
open to discussing a good name here.  We use the term 'expand' in other places (like 'ExpandNullable').  Maybe DoNotExpandTuples?